workers/loc.api/sync/data.inserter/halpers.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+
+const searchClosePrice = async (
+  dao,
+  {
+    auth,
+    symbol,
+    end,
+    id
+  }
+) => {
+  const trades = await dao.findInCollBy(
+    '_getTrades',
+    {
+      auth,
+      params: {
+        symbol,
+        end
```

Comment:
As an idea we can add a limit as to not have a very big response that consumes memory